services/madoc-ts/src/routes/projects/list-project-assignee-stats.ts

```diff
@@ -0,0 +1,39 @@
+import { RouteMiddleware } from '../../types/route-middleware';
+import { api } from '../../gateway/api.server';
+import { userWithScope } from '../../utility/user-with-scope';
+import { extractIdFromUrn } from '../../utility/parse-urn';
+
+export const listProjectAssigneeStats: RouteMiddleware = async context => {
+  const { siteId } = userWithScope(context, ['site.admin']);
```

Comment:
I think this should be a site endpoint - so it will work for users that are not logged in. If you check the routes/sites there should be an example of how to get the siteApi that you can use to make requests.